webpack/middleware/questions_from_forms.js

```diff
@@ -0,0 +1,29 @@
+import {
+  FETCH_FORM_FULFILLED,
+  FETCH_FORMS_FULFILLED,
+  FETCH_QUESTIONS_FULFILLED
+} from '../actions/types';
+
+const questionsFromForms = store => next => action => {
+  switch (action.type) {
+    case FETCH_FORMS_FULFILLED:
+      const forms = action.payload.data;
+      form.forEach((f) => {
```

Comment:
`form` is not defined. I think you want `forms`. This middleware needs a test to catch this. It breaks the index page.